MFVNeutralino/plugins/EventProducer.cc

```diff
@@ -418,7 +442,7 @@ void MFVEventProducer::produce(edm::Event& event, const edm::EventSetup& setup)
 
     // match trigger and offline jets
     mevent->jet_hlt_push_back(jet, triggerfloats->hltpfjets, false);
-    mevent->jet_hlt_push_back(jet, triggerfloats->hltcalojets, true);
+    mevent->jet_hlt_push_back(jet, triggerfloats->hltdisplacedcalojets, true);
 
```

Comment:
Maybe a @shaunhogan question, don't we ultimately use calo jets for some btag triggers, not just the displaced jet ones? Maybe they are distinct sets and both are needed though.....?